src/snowflake/connector/antlr/sql_separator.py

```diff
@@ -0,0 +1,194 @@
+#
+# Copyright (c) 2012-2021 Snowflake Computing Inc. All rights reserved.
+#
+
+#
+# Copyright (c) 2012-2021 Snowflake Computing Inc. All right reserved.
+#
+
+from antlr4 import CommonTokenStream, InputStream, ParseTreeWalker
+
+from .querySeparatorLexer import querySeparatorLexer
+from .querySeparatorListener import querySeparatorListener
+from .querySeparatorParser import querySeparatorParser
+
+COMMENT_CHANNEL: int = 9
+
+
+def is_put_or_get(stmt: str):
```

Comment:
You seem to be missing all the type-hints for return statements
```suggestion
def is_put_or_get(stmt: str) -> bool:
```